README.md

```diff
@@ -29,6 +29,7 @@ You may [restart the session or reboot the watch](https://asteroidos.org/wiki/us
 | ![thumbnail](.thumbnails/analog-asteroid-logo-round.webp) |![thumbnail](.thumbnails/analog-asteroid-logo.jpg) | [analog-asteroid-logo](analog-asteroid-logo/usr/share/asteroid-launcher/watchfaces/analog-asteroid-logo.qml) | [eLtMosen](https://github.com/eLtMosen) |
 | ![thumbnail](.thumbnails/analog-aviator-round.webp) |![thumbnail](.thumbnails/analog-aviator.jpg) | [analog-aviator](analog-aviator/usr/share/asteroid-launcher/watchfaces/analog-aviator.qml) | [eLtMosen](https://github.com/eLtMosen) |
 | ![thumbnail](.thumbnails/analog-classy-roman-round.webp) | ![thumbnail](.thumbnails/analog-classy-roman.jpg) | [analog-classy-roman](analog-classy-roman/usr/share/asteroid-launcher/watchfaces/analog-classy-roman.qml) | [eLtMosen](https://github.com/eLtMosen) |
+| ![thumbnail](.thumbnails/analog-commander-round.webp) | ![thumbnail](.thumbnails/analog-commander.webp) | [analog-commander](analog-commander/usr/share/asteroid-launcher/watchfaces/analog-commander.qml) | [IvoHulsman](https://github.com/ivohulsman) |
```

Comment:
The round thumbnail is fine, but you should choose a wallpaper for the square watchface.  Ideally, it would be one of the standard ones.